pkg/importer/dataStream.go

```diff
@@ -183,6 +212,38 @@ func (d dataStream) http() (io.ReadCloser, error) {
 	return resp.Body, nil
 }
 
+func (d dataStream) glance() (io.ReadCloser, error) {
```

Comment:
Maybe in a separate pr, but we should be consistent on how we define dataStream methods. Sometimes we pass a ptr to dataStream even when the method does not modify the receiver. I probably started this inconsistency by forgetting that the 512 byte slice added to dataStream{} is only taking up the room for the slice header and not the full 512 bytes.